app/ledger.js

```diff
@@ -168,7 +168,8 @@ const doAction = (action) => {
       break
 
     case appConstants.APP_ON_CLEAR_BROWSING_DATA:
-      if (action.clearDataDetail.get('browserHistory') && !getSetting(settings.PAYMENTS_ENABLED)) reset(true)
+      // TODO
```

Comment:
I was thinking to add state directly in same as we do with other reducers. One thing that we can also do is get state only in this case with `getState`